python-frontend/src/main/java/org/sonar/python/semantic/v2/TypeInferenceV2.java

```diff
@@ -61,10 +62,15 @@ public TypeInferenceV2(TypeTable projectLevelTypeTable, PythonFile pythonFile, S
   }
 
   public Map<SymbolV2, Set<PythonType>> inferTypes(FileInput fileInput) {
-    TrivialTypeInferenceVisitor trivialTypeInferenceVisitor = new TrivialTypeInferenceVisitor(projectLevelTypeTable, pythonFile, fullyQualifiedModuleName);
+    WildcardImportVisitor wildcardImportVisitor = new WildcardImportVisitor(projectLevelTypeTable);
```

Comment:
Why would this need to be its dedicated visitor?
I know we're going for a naive implementation, but I think it's still sensible to assume that we don't want the following resolution:
```
foo()
from lib import * # contains foo
```

I'm under the impression that relying on the main visit of `TrivialTypeInferenceVisitor` should be enough here, no?

Reply:
The `TrivialTypeInferenceVisistor` is enough (initially that's what we did). Our thought was that if it is moved to its own visitor, it is easier to rip it out when we eventually will do that. But we didn't think of the example you provided